packages/react-composites/src/composites/ChatComposite/file-sharing/AttachmentUpload.ts

```diff
@@ -95,7 +95,7 @@ interface ClearAction extends Action {
 /**
  * @private
```

Comment:
Should we change the access modifier from`private` to something else as it's exported right now?